lib/linter-registry.js

```diff
@@ -1,12 +1,17 @@
 /* @flow */
+/* eslint-disable import/no-duplicates */
 
+import ConfigFile from 'sb-config-file'
 import { Emitter, CompositeDisposable } from 'atom'
 import type { TextEditor, Disposable } from 'atom'
-import { linter as validateLinter, messages as validateMessages, messagesLegacy as validateMessagesLegacy } from './validate'
-import { getEditorCursorScopes, normalizeMessages, normalizeMessagesLegacy, isPathIgnored, shouldTriggerLinter, $version, $activated, $requestLatest, $requestLastReceived } from './helpers'
+
+import * as Helpers from './helpers'
+import * as Validate from './validate'
+import { $version, $activated, $requestLatest, $requestLastReceived, LINTER_CONFIG_FILE_PATH, LINTER_CONFIG_FILE_DEFAULT } from './helpers'
```

Comment:
This particular type of error is actually annoying to fix with `eslint-disable-next-line` comment, because you have to do it twice on both imports instead of once so I just disabled it in this file entirely